dev/TreeView/InteractionTests/TreeViewTests.cs

```diff
@@ -853,17 +853,25 @@ private void TreeViewDragAndDropBetweenNodes(bool isContentMode = false)
                 // 5% from the edge of the next item 
                 var distance = (int)(height * 1.45);
 
+                Log.Comment("Click on the item to help make the drag more reliable");
+                dragUIObject.Click();
+
                 Log.Comment("Starting Drag...distance:" + distance);
-                InputHelper.DragDistance(dragUIObject, distance, Direction.South);
+                InputHelper.DragDistance(dragUIObject, distance, Direction.South, 4000);
```

Comment:
The metadata says that 4000 is the default

Reply:
Indeed, removed.